### PR TITLE
Integrate search bar

### PR DESCRIPTION
**Pull request by Dagurf21**

### CODE REVIEW

File: src/views/views.py [hunk 1]
```diff
@@ -71,13 +71,27 @@ def home(request):
...
+
     return GeneralViewController(request).render('pages/home.html', {
         'job_offers': job_offers,
         'companies_list': companies_list,
         'company_filter': filters.get('company'),
         'order_filter': filters.get('order_by'),
+        'query': query,
     })
 
+def home_search(request):
```
Comment (milosptr) [inline src/views/views.py]: @Dagurf21 what's this tho?

**Review by milosptr — CHANGES_REQUESTED** [review]

File: src/views/views.py (hunk 1 above)
Comment (Dagurf21) [reply src/views/views.py]: Started doing a new view for the search handling, didnt need it just added it to def(home) view. Can be deleted

Comment (milosptr) [reply src/views/views.py]: Can you delete it so I can merge this? 🙏

**Review by milosptr — CHANGES_REQUESTED** [review]

**Comment by Dagurf21** [comment]
Should be done

**Pull request merged**